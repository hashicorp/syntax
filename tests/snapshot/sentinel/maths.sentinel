# https://developer.hashicorp.com/sentinel/docs/language/arithmetic
value = 4 + 8
value = 8 * 2
value = 8 / 4
value = 8 / 5
value = 8 % 5
