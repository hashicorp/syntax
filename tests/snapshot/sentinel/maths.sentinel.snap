># https://developer.hashicorp.com/sentinel/docs/language/arithmetic
#^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.sentinel comment.line.number-sign.sentinel
>value = 4 + 8
#^^^^^^ source.sentinel
#      ^ source.sentinel keyword.operator.symbol.sentinel
#       ^ source.sentinel
#        ^ source.sentinel constant.numeric.number.sentinel
#         ^ source.sentinel
#          ^ source.sentinel keyword.operator.symbol.sentinel
#           ^ source.sentinel
#            ^ source.sentinel constant.numeric.number.sentinel
>value = 8 * 2
#^^^^^^ source.sentinel
#      ^ source.sentinel keyword.operator.symbol.sentinel
#       ^ source.sentinel
#        ^ source.sentinel constant.numeric.number.sentinel
#         ^ source.sentinel
#          ^ source.sentinel keyword.operator.symbol.sentinel
#           ^ source.sentinel
#            ^ source.sentinel constant.numeric.number.sentinel
>value = 8 / 4
#^^^^^^ source.sentinel
#      ^ source.sentinel keyword.operator.symbol.sentinel
#       ^ source.sentinel
#        ^ source.sentinel constant.numeric.number.sentinel
#         ^ source.sentinel
#          ^ source.sentinel keyword.operator.symbol.sentinel
#           ^ source.sentinel
#            ^ source.sentinel constant.numeric.number.sentinel
>value = 8 / 5
#^^^^^^ source.sentinel
#      ^ source.sentinel keyword.operator.symbol.sentinel
#       ^ source.sentinel
#        ^ source.sentinel constant.numeric.number.sentinel
#         ^ source.sentinel
#          ^ source.sentinel keyword.operator.symbol.sentinel
#           ^ source.sentinel
#            ^ source.sentinel constant.numeric.number.sentinel
>value = 8 % 5
#^^^^^^ source.sentinel
#      ^ source.sentinel keyword.operator.symbol.sentinel
#       ^ source.sentinel
#        ^ source.sentinel constant.numeric.number.sentinel
#         ^ source.sentinel
#          ^ source.sentinel keyword.operator.symbol.sentinel
#           ^ source.sentinel
#            ^ source.sentinel constant.numeric.number.sentinel
>